bin/pgadmin4/start-pgadmin4.sh

```diff
@@ -62,7 +69,13 @@ cd ${PGADMIN_DIR?}
 if [[ ! -f /var/lib/pgadmin/pgadmin4.db ]]
 then
     echo_info "Setting up pgAdmin4 database.."
-    python setup.py > /tmp/pgadmin4.stdout 2> /tmp/pgadmin4.stderr
+    which python3
+    if [[ $? == 0 ]]
```

Comment:
```suggestion
    if command -v python3
```

Reply:
Hm this did not work, going to troubleshoot further